mbuild/lib/recipes/alkane.py

```diff
@@ -14,7 +14,7 @@ def __init__(self, n=3, cap_front=True, cap_end=True):
             cap_front: Add methyl group to beginning of chain ('down' port).
             cap_end: Add methyl group to end of chain ('up' port).
         """
-        if n < 2:
+        if n < 1:
```

Comment:
`n < 2` was correct. The error message was wrong. 

In this line:
https://github.com/mosdef-hub/mbuild/blob/584695616488cff18022edc7d07511a5d64adb9a/mbuild/lib/recipes/alkane.py#L26

`n = n - 2` would end up being negative and raise an error.

Reply:
You're right. I also found a few edge cases with this class (like when `n=2, cap_front=False, cap_end=False`), so I will probably work on cleaning this class up a little bit.